scripts/mrbind/install_mrbind_macos.sh

```diff
@@ -43,5 +43,5 @@ export PATH="$HOMEBREW_DIR/opt/llvm@$CLANG_VER/bin:$PATH"
 # Guess the number of build threads.
 [[ ${JOBS:=} ]] || JOBS=$(sysctl -n hw.ncpu)
 
-CC=clang CXX=clang++ cmake -B build
+CC=clang CXX=clang++ cmake -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON
```

Comment:
is it debug info?

Reply:
Nope, just emits a list of source files and compiler commands for them.

Reply:
Nope, this just prints the list of files compiled, and the compilation commands. Some IDEs need this to function correctly.